.github/workflows/release-and-publish.yml

```diff
@@ -0,0 +1,69 @@
+# This is a workflow to create github release for a tag and publish the package
+# to npm repo
+
+name: release-and-publish
+
+# Controls when the action will run. Triggers the workflow on push of tag
+# request events but only tags matching the configured regex
+on:
+  push:
+    tags:
+      # patterns to match for tag creation. Here all tags similar to v1.0 or
+      # v1.2.0 will trigger this action
+      - 'v[0-9]+.[0-9]+.[0-9]+*'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in
+# parallel
+jobs:
+  # This workflow contains a single job called "build"
+  create_release:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the
+    # job
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access
+    # it
+    - name: Checkout code
+      uses: actions/checkout@v2
+
+    # Runs a single command using the runners shell
+    - name: Create release for tag
+      uses: actions/create-release@latest
+      env:
+        # this is provided by github, you don't need to do anything here
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
+        release_name: Release ${{ github.ref }}
```

Comment:
Are these the tag name, release name and body which can be seen in GitHub release page UI ?

Reply:
These are those parameters you provide when releasing through UI. If you would like some other values for them let me know I can update.

Reply:
yes I think we should stay with consistent pattern for `release_name`. In GitHub release page, I always put `release_name` with the same value as `tag_name` so that GitHub release page can display the `Latest` label correctly.

Reply:
Sure. Will make them same.

Reply:
> yes I think we should stay with consistent pattern for `release_name`. In GitHub release page, I always put `release_name` with the same value as `tag_name` so that GitHub release page can display the `Latest release` label correctly and also highlight the release name. I figured out that while playing with GitHub release page.

Can you elaborate more on this? Because on my sample I can still see latest release even though the tag_name and release_name are different. For example: https://github.com/anshulguleria/ts-jest/releases

Reply:
![image](https://user-images.githubusercontent.com/27772165/80638750-38f0fe00-8a61-11ea-9a2a-817979661743.png)
Here is what I meant. I don't know why the label `Latest release` in GitHub matters about tag name and release name, but as long as I make them the same name then the label `Latest release` is attached to the release and the release name is displayed with big font.